app/cells/plugins/core/integer_cell.rb

```diff
@@ -0,0 +1,52 @@
+module Plugins
+  module Core
+    class IntegerCell < Plugins::Core::Cell
+      
+      def input
+        render
+      end
+     
+      private
+      
+      def max
+        field.validations[:max] 
```

Comment:
You have a lot of similarities between this cell and the `FloatCell`, could you refactor the common ground methods like this into a module to use for both and dry up the code